packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.141.0
+------------------------------
+*June 17, 2021*
+
+### Changed
```

Comment:
Fixed?

Reply:
Will fix this later, since we need this merged ASAP and changing this will kick off the whole build progress again.